MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -21,6 +21,8 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		refresh()
 	}
 
+	@IBOutlet public var errorView: ErrorView!
```

Comment:
The setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView!
```